pkg/volume/gitrepo/git_repo_test.go

```diff
@@ -113,14 +113,14 @@ func TestPlugin(t *testing.T) {
 				Name: "vol1",
 				VolumeSource: v1.VolumeSource{
 					GitRepo: &v1.GitRepoVolumeSource{
-						Repository: gitUrl,
+						Repository: gitURl,
```

Comment:
```suggestion
						Repository: gitURL,
```
I think you forgot to capitalize the final L :)

Reply:
thanks, i will modify it.

Reply:
@mattjmcnaughton thanks for your help ,i will modify it.